### ISSUE TITLE
about the pretrained model weights

### ISSUE THREAD

**Issue by boozyguo** [asinghani/freespace-nn#1]
Hello, asinghani. the freespace-nn project is wonderful.  Also, I study the paper “Real-Time Freespace Segmentation on Autonomous Robots for Detection of Obstacles and Drop-Offs" and try the freespace-nn project. 
So, Could you help to provide the pretrained model weights "/home/ubuntu/model-floornet.h5" ? 
Thank you.

**Comment by asinghani** [comment]
Hello @boozyguo. 

Thank you for your interest in the project. I did not release the weights because they were trained on sets of environments that I had access to, but they will not work very well on extremely-different environments. For using this in your own work, you would need to collect data for the specific environment(s) that you are planning on using for your robot, and then train the model using the training scripts available in this repository to get the model weights. Best of luck in your research.

**Comment by boozyguo** [comment]
Thank you @asinghani  for your reply. I will try to trian the model. Also, could you give me some information about the label data?  I wonder the label data are some binary images?  And what tools are used to label the data? Also, how many images used in your training and validation?
Thank you.
